pkg/apis/certificates/types.go

```diff
@@ -122,7 +122,7 @@ type KeyUsage string
 const (
 	UsageSigning            KeyUsage = "signing"
 	UsageDigitalSignature   KeyUsage = "digital signature"
-	UsageContentCommittment KeyUsage = "content committment"
+	UsageContentCommittment KeyUsage = "content commitment"
```

Comment:
This is more than just a comment and may have compatibility impact.  Note also that the const is misspelled also.

This also lines up with the cloudflare library that this string gets passed to: https://sourcegraph.com/github.com/kubernetes/kubernetes@886e04f1fffbb04faf8a9f9ee141143b2684ae68/-/blob/vendor/github.com/cloudflare/cfssl/config/config.go#L460-461.  I'm guessing that we don't want to change this one.